src/NLog/Targets/ConsoleTarget.cs

```diff
@@ -127,6 +135,15 @@ public ConsoleTarget(string name) : this()
         protected override void InitializeTarget()
         {
             PauseLogging = false;
+            if (DetectConsoleAvailable)
+            {
+                PauseLogging = !IsConsoleAvailable();
+                if (PauseLogging && LoggingConfiguration != null)
+                {
+                    foreach (var loggingRule in LoggingConfiguration.LoggingRules)
+                        loggingRule.Targets.Remove(this);
```

Comment:
I can remove this. The most important part is to avoid performing the unnecessary layout-rendring.